tensorflow/contrib/learn/python/learn/datasets/base.py

```diff
@@ -144,3 +144,34 @@ def maybe_download(filename, work_directory, source_url):
         size = f.size()
       print('Successfully downloaded', filename, size, 'bytes.')
   return filepath
+
+def split_into_train_and_test(dataset, test_ratio=0.3, shuffle=False):
```

Comment:
we have this function here - https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/learn/python/learn/estimators/_sklearn.py#L168 if you want to just pull it out from there to here and refer from _sklearn part - would be great.